src/ui/state/layout.ts

```diff
@@ -5,10 +5,13 @@ export type LayoutState = {
   selectedPrimaryPanel: PrimaryPanelName;
   selectedPanel: SecondaryPanelName;
   viewMode: ViewMode;
+  toggleMode: ToggleMode;
   toolboxLayout: ToolboxLayout;
 };
 
 export type ViewMode = "dev" | "non-dev";
+export type ToggleMode = "dev" | "non-dev";
```

Comment:
We can just use `ViewMode` as the type where `ToggleMode` is currently being used.